Library/Homebrew/test/requirements/linux_requirement_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "requirements/linux_requirement"
+
+describe LinuxRequirement do
+  subject(:requirement) { described_class.new }
+
+  describe "#satisfied?" do
+    it "returns true on Linux" do
+      expect(requirement.satisfied?).to eq(OS.linux?)
+    end
+  end
+end
```

Comment:
two questions:
1. shouldn't this be under `Library/Homebrew/test/os/linux`?
1.1 if yes, shouldn't this be called `os_requirement_spec.rb`?

Reply:
Ideally, specs should be in the same directory structure as the main code. Also, in this case, it should also work (and be tested) on macOS, i.e. return `false`. And it should be named after the class being tested.